docs/content/en/integration/openid-connect/budibase/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Budibase"
+description: "Integrating Budibase with the Authelia OpenID Connect 1.0 Provider."
+lead: ""
+date: 2023-11-12T15:50:35+00:00
+draft: false
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 620
+toc: true
+community: true
+---
+
+## Tested Versions
+
+- [Authelia]
+  - [v4.37.5](https://github.com/authelia/authelia/releases/tag/v4.37.5)
+- [Budibase]
+  - 2.13.9
+
+## Before You Begin
+
+{{% oidc-common %}}
+
+### Authelia
+
+Authelia configuration.yml
+
+```yaml
+identity_providers:
+  oidc:
+    clients:
+      - id: budibase
+        secret: mysecret
+        authorization_policy: two_factor
+        redirect_uris:
+          - https://budibase.example.com/api/global/auth/oidc/callback
+        scopes:
+          - openid
+          - profile
+          - email
+          - offline_access
```

Comment:
```suggestion
      - id: 'budibase'
        description: 'Budibase'
        secret: '$pbkdf2-sha512$310000$c8p78n7pUMln0jzvd4aK4Q$JNRBzwAo0ek5qKn50cFzzvE9RXV88h1wJn5KGiHrD0YKtZaR/nCb2CJPOsKaPK0hjf.9yHxzQGZziziccp6Yng'  # The digest of 'insecure_secret'.
        authorization_policy: 'two_factor'
        redirect_uris:
          - 'https://budibase.example.com/api/global/auth/oidc/callback'
        scopes:
          - 'openid'
          - 'profile'
          - 'email'
          - 'offline_access'
        token_endpoint_auth_method: 'client_secret_basic'
```

The token_endpoint_auth_method will need to be changed to `client_secret_post` if that's the method they use for client auth.